mapbox/libandroid-services/src/main/java/com/mapbox/services/android/location/LostLocationEngine.java

```diff
@@ -89,9 +89,9 @@ public Location getLastLocation() {
   public void requestLocationUpdates() {
     // Common params
     LocationRequest request = LocationRequest.create()
-      .setInterval(1000)
-      .setFastestInterval(1000)
-      .setSmallestDisplacement(3.0f);
+      .setInterval(interval)
```

Comment:
@cammace I like where this is going, particularly the great javadoc added. I'd like to make this fields optional to set, like GMS does ([example](https://developers.google.com/android/reference/com/google/android/gms/location/LocationRequest.html#setFastestInterval(long)). How about we make them objects instead with a default value of `null`?